src/ServicePulse.Host/vue/src/components/configuration/RetryRedirectEdit.vue

```diff
@@ -175,7 +173,7 @@ function save() {}
 }
 
 .modal-container {
-  width: 400px;
+  width: 600px;
```

Comment:
since this CSS is unscoped, the order of import into its parent component matters. Previously, ConfirmDialog was imported later, so it's equivalent style with 600px width took precedence. This forces the width to be 600 again, and should be scoped at a later date when scoped styles are implemented thoroughly throughout the codebase.